machine_management/index.adoc

```diff
@@ -31,12 +31,22 @@ As a cluster administrator, you can perform the following actions:
 
 * Create a compute machine set for the following cloud providers:
 
+** xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-alibaba[Alibaba]
+
 ** xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-aws[AWS]
 
+** xref:../machine_management/creating_machinesets/creating-machineset-aws.adoc#creating-machineset-azure-stack-hub[Azure Stack Hub]
```

Comment:
I would recommend moving Azure Stack Hub after "regular" Azure on the list. Also, link update:
```suggestion
** xref:../machine_management/creating_machinesets/creating-machineset-azure-stack-hub.adoc#creating-machineset-azure-stack-hub[Azure Stack Hub]
```